modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
```

Comment:
See OpenShift documentation guidelines for [module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)
```suggestion
:_mod-docs-content-type: REFERENCE
```

Reply:
ACK.

Addressed